### ISSUE TITLE
I think key-mon should be aware of accessibility feature such as sticky keys.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [scottkirkwood/key-mon#59]
```
I often use this feature but it seems not reflected in key-mon.
```

Original issue reported on code.google.com by `irsyad.l...@gmail.com` on 22 Sep 2010 at 9:16
